setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = avionmqtt
-version = 1.3.9
+version = 1.4.0
```

Comment:
Let's just bump this to 1.3.10 (though there's not really a system to the increments right now).

Reply:
Sounds good, wasn't sure if you wanted to go to 10 or not